pkg/ansible/runner/runner_test.go

```diff
@@ -26,32 +26,20 @@ import (
 	"github.com/operator-framework/operator-sdk/pkg/ansible/watches"
 )
 
-func playbookCmd(path string, ident string, inputDirPath string, maxArtifacts int) *exec.Cmd {
-	return exec.Command("ansible-runner", "-vv", "--rotate-artifacts", fmt.Sprintf("%v", maxArtifacts), "-p", path, "-i", ident, "run", inputDirPath)
-}
-
-func roleCmd(path string, ident string, inputDirPath string, maxArtifacts int) *exec.Cmd {
-	rolePath, roleName := filepath.Split(path)
-	return exec.Command("ansible-runner", "-vv", "--rotate-artifacts", fmt.Sprintf("%v", maxArtifacts), "--role", roleName, "--roles-path", rolePath, "--hosts", "localhost", "-i", ident, "run", inputDirPath)
-}
-
-type cmdFuncType func(ident, inputDirPath string, maxArtifacts int) *exec.Cmd
-
-func checkCmdFunc(t *testing.T, cmdFunc cmdFuncType, role, playbook string, watchesMaxArtifacts, runnerMaxArtifacts int) {
+func checkCmdFunc(t *testing.T, cmdFunc cmdFuncType, playbook, role string, verbosity int) {
 	ident := "test"
 	inputDirPath := "/test/path"
-	var path string
+	maxArtifacts := 1
```

Comment:
Since this is an argument to the `cmdFunc` it was silly to do an expected vs actual...just check that given the arguments `ident`, `inputDirPath`, and `maxArtifacts` that the `cmdFunc` on the created runner matches what we expect based on the particular watch under test.